CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 3.0.1
+
+- [PR #37](https://github.com/alphagov/govuk_message_queue_consumer/pull/37)
```

Comment:
Perhaps the changelog could be more succinct - eg "Fix a bug that caused the test-helpers to error (#34)". I think it's meant to be a short note that is useful to determine if the user should upgrade.